package/gluon-mesh-batman-adv/src/respondd-statistics.c

```diff
@@ -299,10 +320,17 @@ static struct json_object * get_clients(void) {
 		},
 	};
 
+	if (getifaddrs(& opts.ifaddr) == -1) {
+		opts.ifaddr = NULL;
+	}
```

Comment:
Single statements are not wrapped in braces in this file. So remove them.

Reply:
I don't think this is a good idea in general.

Reply:
I am indifferent on this but I tend to go with @blocktrron for consistency in this file for now.